docs/docs/using-cloudinary-image-service.md

```diff
@@ -0,0 +1,156 @@
+- **`resourceType`** **:** This is the resource type of the media assets - either an image or a video.
+- **`prefix`** **:** This is the folder (in your Cloudinary account) in which the files reside. In the example above, the folder is called `gatsby-source-cloudinary`. Assign a name of your choice.
+  Other optional options are `type`, `tags`, and `maxResult`.
+
+Here's a [link to the README](https://github.com/Chuloo/gatsby-source-cloudinary#query-parameters) for more information.
+
+## Gatsby-transformer-cloudinary
+
+After sourcing media files from Cloudinary, you will be able to leverage Cloudinary’s media transformation capabilities. To do so, use `gatsby-transformer-cloudinary` which is a type of [transformer plugin](/docs/creating-a-transformer-plugin/) that is used to change image formats, styles and dimensions. It also optimizes images for minimal file size alongside high visual quality for an improved user experience and minimal bandwidth.
+
+Here's a [demo site that uses the gatsby-transformer-plugin](https://gatsby-transformer-cloudinary.netlify.com/fluid/)
+
+### Prerequisites
+
+Before using the `gatsby-transformer-cloudinary` plugin you should do the following:
+
+- Upload your images to a folder on Cloudinary. This folder can have any name of your choosing.
+- Have the gatsby-source-cloudinary plugin to get images.
+- Obtain your API key and API secret from your Cloudinary dashboard.
+- Have the [dotenv](https://www.npmjs.com/package/dotenv) module installed for loading environment variables from a `.env` file.
+
+### Using gatsby-transformer-cloudinary
+
+1. Install `gatsby-transformer-cloudinary` and `gatsby-source-filesystem` which creates the File nodes that the Cloudinary transformer plugin works on.
+
+```shell
+ npm install gatsby-transformer-cloudinary gatsby-source-filesystem
+```
+
+2. In the root of your project, create an environment file called `.env` to which to add your Cloudinary credentials and their values.
+
+```
+CLOUDINARY_API_KEY=xxxxxxxxxxxxxx
+CLOUDINARY_API_SECRET=xxxxxxxxxxxxxxxxxxxx
+CLOUDINARY_CLOUD_NAME=xxxxx
+```
+
+3. Configure `gatsby-config.js`
+
+```js:title=gatsby-config.js
+require('dotenv').config({
+  path: `.env.${process.env.NODE_ENV}`,
+});
+
+module.exports = {
+  plugins: [
+    {
+      resolve: `gatsby-source-filesystem`,
+      options: {
+        name: `images`,
+        path: `${__dirname}/src/images`,
+      },
+    },
+    {
+      resolve: 'gatsby-transformer-cloudinary',
+      options: {
+        cloudName: process.env.CLOUDINARY_CLOUD_NAME,
+        apiKey: process.env.CLOUDINARY_API_KEY,
+        apiSecret: process.env.CLOUDINARY_API_SECRET,
+
+        // This folder will be created if it doesn’t exist.
+        uploadFolder: 'gatsby-cloudinary',
+
+    },
+  ],
+};
+```
+
+> In `gatsby-config.js`, responsive breakpoints can be created for each image, use the fluidMaxWidth and fluidMinWidth options to set them.
```

Comment:
Last outstanding question I have is whether we have a link for more information? (maybe the README?)

Reply:
Well, these parameters are just to show how to add responsive breakpoints there is no further information after that. I did, however, link the README from npm for context.